### PR TITLE
[REG-1024] - Allow ENS domains renewal for non-parked domains

### PR DESCRIPTION
**Pull request by DeRain**
### 1. Contracts versioning
- [x] Make sure that the `patch` version of the contracts is increased if changes have been made to the `UNSRegistry`, `MintingManager` or `ProxyReader` contracts.
- [ ] Make sure that the `minor` version of the contracts is increased if breaking changes have been made to the `UNSRegistry`, `MintingManager` or `ProxyReader` contracts. It includes changes of interfaces.
### 2. Contracts licensing
- [x] Make sure that no **SPDX-License-Identifier** defined in contracts.
- [x] Make sure that the **header** is added to the new contract files. 
  ```
  // @author Unstoppable Domains, Inc.
  // @date {Month} {Day}(ordinal), {Year}
  ```
### 3. Coverage
- [x] Make sure that the coverage of contracts has not decreased and strive **100%**
### 4. Configs versioning
- [ ] Make sure that the version of `uns-config.json` is increased if changes have been made to the config.
- [ ] Make sure that the version of `resolver-keys.json` is increased if changes have been made to the config.
### 5. Package versioning
- [x] Make sure that the `patch` version of package is increased if valuable changes have been made to the package. It includes contracts update, configs update, etc.
- [ ] Make sure that the `major.minor` version of package is synced with version of `UNSRegistry` contract.
- [x] Make sure that the `CHANGELOG` is updated with short description for the new version. 
### 6. Code review
- [ ] `resolver-keys.json` code review is required from **DevTools** team
- [x] For all other changes code review is required from **Registry** team

### CODE REVIEW

File: contracts/custody/ENSCustody.sol [hunk 1]
```diff
@@ -157,12 +157,6 @@ contract ENSCustody is
     }
 
     function renew(string calldata name, uint256 duration) external onlyMinter nonReentrant {
-        INameWrapper _wrapper = INameWrapper(StorageSlotUpgradeable.getAddressSlot(_ENS_WRAPPER_SLOT).value);
```
Comment (nickshatilo) [inline contracts/custody/ENSCustody.sol:160]: Shall we check if the token actually exists on ENS?

Comment (DeRain) [reply contracts/custody/ENSCustody.sol:160]: @nickshatilo ENS checks on their side. Don't see any benefits cheking this on our side as well.

Comment (DeRain) [reply contracts/custody/ENSCustody.sol:160]: @nickshatilo I'll add test to cover this case

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "uns",
-  "version": "0.8.16",
+  "version": "v0.8.17",
```
Comment (aquiladev) [inline package.json:3]:
```suggestion
  "version": "0.8.17",
```

File: test/custody/ENSCustody.test.ts [hunk 3]
```diff
@@ -367,28 +401,22 @@ describe('ENSCustody', function () {
...
-      await registerName(name);
-
-      await expect(custody.connect(minter).renew(name, REGISTRATION_TIME)).to.be.revertedWith('UnknownToken');
-    });
-
     it('should revert renewing of expired domain', async () => {
       const name = 'ts-np12';
       await topupCustody(name);
       await registerAndParkName(name, minter, ZERO_ADDRESS, false);
+      await topupCustody(name, REGISTRATION_TIME);
```
Comment (aquiladev) [inline test/custody/ENSCustody.test.ts:413]: why do we need topup twice?

Comment (aquiladev) [reply test/custody/ENSCustody.test.ts:413]: I removed it locally, nothing changed. coverage the same 🤔

**Pull request merged**